bolt-modules/boltlib/lib/puppet/functions/apply_prep.rb

```diff
@@ -13,10 +13,12 @@
 # > **Note:** Not available in apply block
 Puppet::Functions.create_function(:apply_prep) do
   # @param targets A pattern or array of patterns identifying a set of targets.
+  # @param options Options hash. For now only supports `required_modules` as an array of modules to sync
```

Comment:
```suggestion
  # @param options Options hash.
  # @option options [Array] _required_modules An array of modules to sync to the target.
```

This will let puppet-strings generate documentation similar to other Bolt functions with options hashes.

Other than this and the change Lucy mentioned above, this looks ready to go.